node/src/indexer/real.rs

```diff
@@ -29,7 +29,13 @@ pub fn spawn_real_indexer(
             let indexer =
                 near_indexer::Indexer::new(indexer_config.to_near_indexer_config(home_dir.clone()))
                     .expect("Failed to initialize the Indexer");
-            let respond_config = load_respond_config_file(&home_dir).unwrap();
+            let respond_config = load_respond_config_file(&home_dir).unwrap_or_else(|err| {
```

Comment:
I'd prefer if load_respond_config_file returned a Result<Option<...>> where if the file doesn't exist it is explicitly returned as an Option::None. The reason is that it can be easy to make a mistake in the yaml syntax and cause the config to fail to deserialize, and yet it would almost silently fall back to single account.

But we can leave that for later.